app/assets/scss/_support-page.scss

```diff
@@ -0,0 +1,8 @@
+.support-page {
+  @media (min-width: 641px) {
+    .page-heading-smaller {
+      margin-bottom: 20px;
```

Comment:
Should this be part of the toolkit heading style?  Why would we only want this margin for this particular page and not for all headings that are this size?
  
I'm really wary of adding per-app styles, especially for something as common as a heading.  We should add styles in the toolkit to ensure consistency across the site.

Reply:
This specific design change is just for this page, everywhere else they want to keep it the same